### PR TITLE
prow: use our own coverage image

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

Fix tektoncd/pipeline#3512

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -1041,7 +1041,7 @@ postsubmits:
     decorate: true
     spec:
       containers:
-      - image: gcr.io/knative-tests/test-infra/coverage:latest # we may want ours here
+      - image: gcr.io/tekton-releases/dogfooding/coverage:latest # we may want ours here
```
Comment (sbwsg) [inline prow/config.yaml:1044]: nit: remove comment?

Comment (vdemeester) [reply prow/config.yaml:1044]: oh I thought I did :swimmer:

**Review by sbwsg — APPROVED** [review]

**Comment by pritidesai** [comment]
thanks @vdemeester 
/lgtm

**Pull request merged**